tests/testthat/test-template.R

```diff
@@ -0,0 +1,14 @@
+test_that("generic report knits", {
+  # temporary directory deleted after test_that block
+  foo <- withr::local_tempdir()
+  create_visc_project(foo, interactive = FALSE)
+  # temporarily setwd() during test_that block
+  withr::local_dir(foo)
+  use_visc_report(report_type = "generic", interactive = FALSE)
+  expect_no_error(
+    rmarkdown::render(file.path('PT-Report', 'PT-Report.Rmd'), quiet = TRUE)
+  )
+  expect_true(
+    file.exists(file.path("PT-Report", "PT-Report.pdf"))
+  )
```

Comment:
Could we also add an expect_snapshot_file() call (see https://testthat.r-lib.org/articles/snapshotting.html#whole-file-snapshotting)? It looks like we can use this to save the pdf, and then it will automatically be uploaded as a GitHub artifact during the CI checks [here](https://github.com/FredHutch/VISCtemplates/actions/runs/9618562082/job/26532718862#step:8:169) so that the pdf can be reviewed on GitHub which would be really nice!